src/System.Private.CoreLib/shared/Interop/Unix/Interop.Libraries.cs

```diff
@@ -2,14 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-
 internal static partial class Interop
 {
     internal static partial class Libraries
     {
         internal const string CoreLibNative = "System.Private.CoreLib.Native";
```

Comment:
System.Private.CoreLib.Native is CoreRT specific. It should not be in the shared Interop.Libraries.cs. There should be two Interop.Libraries.cs: the shared one that has System.Globalization.Native and System.Native; and the unshared one that has System.Private.CoreLib.Native.